### DISCUSSION TITLE
`CodeCompanionChat Add` should do #buffer in normal mode

### DISCUSSION THREAD

**Discussion by edshamis** [olimorris/codecompanion.nvim#525] [Ideas]
Now `CodeCompanionChat Add` in normal mode adds an empty code block.

**Comment by olimorris** [comment]
Yes. As the README states:

> CodeCompanionChat Add - Add visually selected chat to the current chat buffer

It's not intended for normal mode.

**Comment by olimorris** [comment]
Also why should a command append `#buffer` to the chat buffer? That feels very inefficient when you could do `:CodeCompanionChat #buffer what does this file do?`
